### DISCUSSION TITLE
Move to option

### DISCUSSION THREAD

**Discussion by healBvdb** [RickyDane/CoDriver#42] [Ideas]
I was looking for a fast file explorer at work (W11). One thing that I appreciate in my linux config at home is the "Move to" option. I am missing this feature in W11 file explorer. 
It could be nice to add the Move option in the context menu.
In dual pane mode, this could be straightforward in single pane mode it has to be designed

**Comment by RickyDane** [comment]
@healBvdb

So you mean the function that allows you to simply move files or folders from one location to another?
That could easily be implemented.

- For the dual pane view it could be LShift + F5 to move from one pane to the other.
- For the single pane view I could add a "cut" option to prepare the file to be moved from one location to another.
  - This is currently already possible by selecting the item and pressing ctrl + x to cut and ctrl + v to paste.

Let me know, what you think about it.

**Reply by healBvdb** [reply]
In linux I am often using Move To and Copy To.  With file or folder selected it open a new window to select a path.
![image](https://github.com/RickyDane/rdpFX/assets/51165870/9884ea57-4c86-43ea-9d79-07c4f474f85c)

- For the dual pane view it could be LShift + F5 to move from one pane to the other - Yes it works fine
- For single pane view - cut option is OK, but I find it more handy to be able to select a path

**Reply by RickyDane** [reply]
I will work it out and implement the "move to" option with opening a new window like you mentioned!

**Reply by RickyDane** [reply]
I implemented the "move to" option but first want to test it on windows. I just implemented it on my MacBook, cause I'm not at home. As soon as I am able to test and validate it on windows and linux, I will publish a new release.

**Comment by RickyDane** [comment]
Will be closed with #53
